docs/docs/spark-procedures.md

```diff
@@ -504,9 +504,9 @@ Rewrite the manifests in table `db.sample` and align manifest files with table p
 CALL catalog_name.system.rewrite_manifests('db.sample');
 ```
 
-Rewrite the manifests in table `db.sample` and disable the use of Spark caching. This could be done to avoid memory issues on executors.
```

Comment:
we can add this part to the table for the `use-caching` argument.

```
Enabling caching can increase memory footprint on executors.
```

Reply:
Thanks for the suggestion, let me add this description.